src/omnicore/rpc.cpp

```diff
@@ -196,6 +196,36 @@ UniValue omni_getuniquetokens(const UniValue& params, bool fHelp)
     return response;
 }
 
+// locates the owner of a token/range of tokens
+UniValue omni_getuniquetokenowner(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() != 2)
+        throw runtime_error(
+            "omni_getuniquetokenowner propertyid uniquetokenid\n"
+            "\nReturns the owning address for a unique token.\n"
+            "\nArguments:\n"
+            "1. propertyid           (number, required) the property identifier\n"
+            "2. uniquetokenid        (number, required) the token identifier\n"
```

Comment:
Hm.. we're using tokens and properties sometimes for one another. E.g. "token id" == "property id". This could be a bit confusing. Could be changed to "the unique token identifier" at minimum. :)

Reply:
Yeah I see what you mean - I've made the suggested change in 5ff26e50cc8fa73d76305ddc58c762b8c2bd3310 but if you have any thoughts on better wording I'm all ears :)